socorro/scripts/add_crashid_to_queue.py

```diff
@@ -0,0 +1,121 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+
+import pika
+
+from socorro.lib.ooid import is_crash_id_valid
+
+
+EPILOG = """
+To use in a docker-based local dev environment:
+
+    python scripts/add_crashid_to_queue.py socorro.normal <CRASHID>
+
+To use in -prod:
+
+    envconsul -once -prefix socorro/common -prefix socorro/processor env | \
+        /path/to/python scripts/add_crashid_to_queue.py socorro.submitter <CRASHID>
+
+Queues:
+
+* socorro.normal - normal processing
+* socorro.priority - priority processing
+* socorro.submitter (-prod only) - sends crash to -stage environment
+
+"""
+
+
+def get_envvar(key, default=None):
+    if default is None:
+        return os.environ[key]
+    return os.environ.get(key, default)
+
+
+def build_pika_connection(host, port, virtual_host, user, password):
+    """Build a pika (rabbitmq) connection"""
+    return pika.BlockingConnection(
+        pika.ConnectionParameters(
+            host=host,
+            port=port,
+            virtual_host=virtual_host,
+            connection_attempts=10,
+            socket_timeout=10,
+            retry_delay=1,
+            credentials=pika.credentials.PlainCredentials(
+                user,
+                password
+            )
+        )
+    )
+
+
+class WrappedTextHelpFormatter(argparse.HelpFormatter):
+    def _fill_text(self, text, width, indent):
+        """Wraps text like HelpFormatter, but doesn't squash lines
+
+        This makes it easier to do lists and paragraphs.
+
+        """
+        parts = text.split('\n')
+        for i, part in enumerate(parts):
+            parts[i] = super(WrappedTextHelpFormatter, self)._fill_text(part, width, indent)
+        return '\n'.join(parts)
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(
+        formatter_class=WrappedTextHelpFormatter,
+        prog='add_crashid_to_queue.py',
```

Comment:
I'm not familiar with `prog` in argparse but couldn't this be `os.path.basename(__file__)` so we don't repeat the name "twice"?